service/templates/task.html

```diff
@@ -318,46 +324,50 @@ <h4 class="text-center text-muted">No test results to show</h4>
               <tr>
                 <th class="col-lg-6">Test</th>
                 <th class="col-lg-3">Time</th>
-                <th class="col-lg-3">Logs</th>
+                <th class="col-lg-3">Test Logs</th>
+                <th ng-if="task.display_only" class="col-lg-3">Other Logs</th>
```

Comment:
I'd call this "Execution Task"

Reply:
I changed it to have both to make it more obvious that logs can be found there